lib/secrets.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Secrets do
+  @moduledoc false
+
+  use Bitwise
+
+  def secret_add(num) do
+    fn arg -> arg + num end
+  end
+
+  def secret_subtract(num) do
+    fn arg -> arg - num end
+  end
+
+  def secret_multiply(num) do
+    fn arg -> arg * num end
+  end
+
+  def secret_divide(num) do
+    fn arg ->
+      arg / num
+      |> trunc
```

Comment:
We use pipes if and only if it has three links in the chain at least. Otherwise, it’s not _really_ a pipe.
```suggestion
      trunc(arg / num)
```